hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/merkle/MerkleNetworkContext.java

```diff
@@ -819,7 +819,7 @@ private String reprOf(final Instant consensusTime) {
         return consensusTime == null ? NOT_AVAILABLE : consensusTime.toString();
     }
 
-    private String stringifiedBlockHashes() {
+    public String stringifiedBlockHashes() {
```

Comment:
Please add the `@VisibleForTesting` annotation here (see comments at `MerkleStakingInfo.getHistoryHash()` below).